app/views/teacher_interface/registration_number/edit.html.erb

```diff
@@ -7,14 +7,30 @@
   <%= f.govuk_text_field :registration_number, label: { size: "l", tag: "h1" } %>
 
   <% if @teaching_authority_other.present? %>
-    <%= raw GovukMarkdown.render(@teaching_authority_other) %>
+    <%= raw GovukMarkdown.render(@application_form.region.teaching_authority_other) %>
+  <% end %>
+
+  <% if @application_form.region.status_check_online? %>
+    <%= raw GovukMarkdown.render(@application_form.region.teaching_authority_status_information) %>
+  <% end %>
+
+  <% if @application_form.region.sanction_check_online? %>
+    <%= raw GovukMarkdown.render(@application_form.region.teaching_authority_sanction_information) %>
   <% end %>
 
   <%= govuk_details(summary_text: "When we might need more information") do %>
     <p>
       If we’re unable to find you using your registration number, we may need to
       contact you for more information as part of your application.
     </p>
+
+    <% if @application_form.region.status_check_online? %>
+      <%= raw GovukMarkdown.render(@application_form.region.teaching_authority_status_information) %>
+    <% end %>
+
+    <% if @application_form.region.sanction_check_online? %>
+      <%= raw GovukMarkdown.render(@application_form.region.teaching_authority_sanction_information) %>
+    <% end %>
```

Comment:
It looks like the code here is duplicating the lines above, but actually we don't need to render the content within the summary/detail component.